Services/Interfaces/telegram_bot_interface/telegram_bot.py

```diff
@@ -116,6 +119,15 @@ def command_help(update, _):
         elif TelegramBotInterface._is_authorized_chat(update):
             update.message.reply_text(interfaces_bots.UNAUTHORIZED_USER_MESSAGE)
 
+    @staticmethod
+    def set_telegram_api_2fa_code(update, _):
+        if TelegramBotInterface._is_valid_user(update):
+            param = TelegramBotInterface.get_command_param("/telegram_2fa_code", update)
+            telegram_api = Services_bases.TelegramApiService.instance()
+            result = interfaces_util.run_in_bot_main_loop(telegram_api.set_telegram_2fa_code(str(int(param) + 1)))
```

Comment:
Can you add a check if the instance is None ?

Reply:
It's a Singleton class so it will create a new instance if there is no.
https://github.com/Drakkar-Software/OctoBot-Commons/blob/master/octobot_commons/singleton/singleton_class.py

Maybe modify the Singleton class by adding a method to only get current instance without instantiate a newly ?

Reply:
Right, and return None if there is no instance